src/agent/src/rpc.rs

```diff
@@ -243,6 +245,32 @@ impl AgentService {
             }
         }
 
+        let linux = oci
+            .linux()
+            .as_ref()
+            .ok_or_else(|| anyhow!("Spec didn't contain linux field"))?;
+        if let Some(devices) = linux.devices() {
+            for specdev in devices.iter() {
+                let dev_major_minor = format!("{}:{}", specdev.major(), specdev.minor());
```

Comment:
micro-optimization: move this assignment to the `if specdev.path().....` block below.

Reply:
Fixed.